node/components/bft/src/chonky_bft/proposal.rs

```diff
@@ -223,6 +221,7 @@ impl StateMachine {
 
         // Update the state machine.
         self.view_number = message.view().number;
+        metrics::METRICS.replica_view_number.set(self.view_number.0);
```

Comment:
Why move the start of the view to here?

Reply:
I'm moving the metrics updates next to the places where the variable that they track is updated. In particular, before this pr one of such places has been missed.